client/src/components/navbar/LogoBar.js

```diff
@@ -12,15 +12,15 @@ export default function LogoBar(props) {
   return ( 
     <>
       <Navbar bg="primary" variant="dark" style={{position: "fixed", top: "0",width: "100%", zIndex:100}}>
-        <Navbar.Brand as={Link} to="/" className='px-5'>
-          <Image alt="" src = {logo} width="30" height="30" className="d-inline-block align-top" roundedCircle/> {' '} Glacier
+        <Navbar.Brand as={Link} to="/" className='px-5' style={{fontSize: 24, fontWeight: 'bold'}}>
+          <Image alt="" src = {logo} width="40" height="40" className="d-inline-block align-top" roundedCircle /> {' '} Glacier
         </Navbar.Brand>
 
 
         <Nav className="ml-auto" variant="pills" defaultActiveKey="/home">
-          <Nav.Link as={Link} to="/">Home</Nav.Link>
-          { isAuthenticated ? <Nav.Link as={Link} to="/sell">Sell</Nav.Link> : ''}
-          { isAuthenticated ? <Nav.Link as={Link} to="/profile">Profile</Nav.Link> : '' }
+          <Nav.Link as={Link} to="/" style={{fontSize: 18, fontWeight: 'bold'}}>Home</Nav.Link>
```

Comment:
Enforcing arbitrary sizes may help make an element more visible in a particular instance, but variable font sizes (e.g. 'em' units) lets the font size adapt to a persons preferences.

Reply:
Okay font sizes have been changed to ems